specifyweb/frontend/js_src/lib/components/AppResources/TabDefinitions.tsx

```diff
@@ -96,6 +97,24 @@ const generateEditor = (xmlSpec: (() => BaseSpec<SimpleXmlNode>) | undefined) =>
       [stateRestored]
     );
     const isReadOnly = React.useContext(ReadOnlyContext);
+
+    const displayWarningPanel = React.useMemo(
+      () => () => {
+        f.maybe(codeMirrorRef.current?.view, openLintPanel);
```

Comment:
can you double-check that this behaves correctly:
- when there are no errors/warnings (Does not do anything and does not steal focus from the editor)
- when there are errors/warnings

Reply:
it does steal focus unfortunately... and if no warnings/errors it's displayed with "No Diagnostic"